CarService.Api/Services/OrderService.cs

```diff
@@ -63,6 +66,28 @@ public OrderService(UserManager<User> userManager, IUnitOfWorkFactory unitOfWork
             }
         }
 
+        public async Task CreateReviewProposition(string email, ReviewPropositionCreationDto reviewPropositionDto)
+        {
+            var user = await _userManager.FindByEmailAsync(email);
+
+            if (user != null)
+            {
+                using (IUnitOfWork unitOfWork = _unitOfWorkFactory.Create())
+                {
+                    IRepository<ReviewProposition> reviewProposition = unitOfWork.Repository<ReviewProposition>();
+                    reviewProposition.Add(new ReviewProposition
+                    {
+                        MechanicId = user.Id,
+                        OrderId = reviewPropositionDto.OrderId,
+                        Comment = reviewPropositionDto.ReviewDescription,
```

Comment:
The comment doesn't equal to Description

Reply:
reviewPropositionDto.ReviewDescription will be changed to reviewPropositionDto.ReviewComment in next commit